onnx/backend/test/data/node/test_gru_with_initial_bias/model.onnx

```diff
@@ -0,0 +1,30 @@
+backend-test:�
```

Comment:
Do we need to check in model.onnx files?

Reply:
yes, that will be needed by backends that will try to run the tests.